Resources/Prototypes/SS220/Maps/centcomm.yml

```
@@ -0,0 +1,14 @@
+- type: gameMap
+  id: SS220_CentComm
+  mapName: 'Центральное Командование'
+  mapPath: /Maps/SS220/ss220-centcomm.yml
+  minPlayers: 10
+  stations:
+    centcomm:
+      stationProto: NanotrasenCentralCommand
+      components:
+        - type: StationNameSetup
+          mapNameTemplate: '{0} Центральное Командование {1}'
+          nameGenerator:
+            !type:NanotrasenNameGenerator
+            prefixCreator: 'TG'
```

```suggestion
            prefixCreator: 'SS220'
```
ну или просто 220, я хз
типа тг это как понимаю отсылка на тгстейшен